.sync.yml

```diff
@@ -42,6 +42,7 @@ Gemfile:
   optional:
     ":development":
     - gem: github_changelog_generator
+      version: '= 1.15.2'
```

Comment:
can you explain why you pin to such an old version? everything below 1.16.4 didnt work well for vox pupuli.